crates/mev-share-sse/src/types.rs

```diff
@@ -26,13 +26,13 @@ pub struct Event {
 #[derive(Debug, Clone, PartialEq, Eq, Hash, Serialize, Deserialize)]
 pub struct EventTransaction {
     /// Transaction recipient address.
-    pub to: Address,
+    pub to: Option<Address>,
     /// 4-byte-function selector
     #[serde(rename = "functionSelector")]
-    pub function_selector: FunctionSelector,
+    pub function_selector: Option<FunctionSelector>,
     /// Calldata of the transaction
     #[serde(rename = "callData")]
-    pub calldata: Bytes,
+    pub calldata: Option<Bytes>,
```

Comment:
so all of these are optional?

should they be serialized as null or skipped?

if skipped, we need a skip_serialize_if annotation

Reply:
should be skipped, updated